src/components/v2/appDetails/k8Resource/utils.tsx

```diff
@@ -0,0 +1,4 @@
+import { Node, NodeFilters } from '@devtron-labs/devtron-fe-common-lib'
+
+export const doesNodeSatisfiesFilter = (node: Node, filter: string) =>
+    node.health?.status.toLowerCase() === filter || (filter === NodeFilters.drifted && node.hasDrift)
```

Comment:
confirm if this shouldn't be in FE lib?

Reply:
we need to check for filters in oss as well